browser-test/src/admin_program_hidden.test.ts

```diff
@@ -0,0 +1,44 @@
+import { startSession, loginAsProgramAdmin, loginAsAdmin, AdminQuestions, AdminPrograms, endSession, logout, loginAsTestUser, selectApplicantLanguage, ApplicantQuestions, userDisplayName } from './support'
+
+describe('Hide a program that should not be public yet', () => {
```

Comment:
Is there any flakiness here? Maybe we should add `await waitForPageJsLoad(page);` in certain places to ensure page loads are complete (following what @leonwong did in https://github.com/seattle-uat/civiform/pull/1617)?